www/index.php

```diff
@@ -5,34 +5,27 @@
 ?>
 
 <h2>What is this site?</h2>
-<p>There are different documentation
-teams operating somewhat independently on PHP.net subprojects. Only a
-few people are involved in more than one documentation project currently,
-and that limits the know-how sharing, which would otherwise be possible and
-desirable. All of the documentation teams use an early fork of the original
-PHP manual build system. These systems evolved independently, and took over
-some improvements from the others on an occasional basis (the PEAR
-documentation team uses a variant of the revcheck developed for phpdoc for
-example). This site is aimed at getting documentation efforts closer, and
-provide more tools and information for manual authors and translators.</p>
+
+<p>
+  This site has instructions and tools for building, translating, and
+  maintaining the 
+  <a href="https://www.php.net/docs.php">PHP Documentation</a>, which
+  is currently availabe in nine languages plus the original English.
```

Comment:
IMO it's clearer, thanks!

nit: I wouldn't specify the exact number, there are a bit more translations (http://doc.php.net/revcheck.php), they are just not displayed on the main site. I'm afraid if some translations become active, or conversely abandoned, we'll forget to update the value here :)

Reply:
Updated this to just say 'multiple'.